### ISSUE TITLE
Favorites are not syncing (?)

### ISSUE THREAD

**Issue by pentool** [HappenApps/Quiver#496]
I can't get the Favorites to sync between two machines. Are they supposed to be "syncable"?
(v3.0.1)

**Comment by ylian** [comment]
No, favorites are not supposed to sync. Could be a feature request if enough people want this.

**Comment by pentool** [comment]
I think that would be nice.

I've noticed the Trash and the Recent items are syncing. Also when I add a new tag to a note, that is syncing, too. So when I mark it as "Favorite", that should prolly sync, too. You might go off to work from a different continent for 2 weeks or so, and all of a sudden... oops, I don't know which ones are my favorites cuz I left them at home on my desktop ;-)

PS: Maybe I'm just imagining it, but the "emptying" of the Trash is not syncing (at least for me). The apps is syncing when I put an item into the Trash. But when I empty the Trash, the contents are not deleted from the other machine.

**Comment by segphault** [comment]
:+1: I'd also like to have favorites sync automatically.

**Comment by VeriCras** [comment]
This feature is required.
hope that additional become.

**Comment by KimMichaelJones** [comment]
I findi t confusing that favorites are not sync'd, please consider adding this.

**Comment by didamaran** [comment]
👍 I'd also like to have favorites sync automatically.

**Comment by kimin-yun** [comment]
👍 I'd also like to have favorites sync automatically. too

**Comment by olibclarke** [comment]
Just noticed this myself - favorites should really sync I think @ylian, this feels like a bug

**Comment by thewongdev** [comment]
Yes, please do add syncing of favorites. It would make it much easier to see favorited (aka pinned) notes on my iPad/iPhone.

**Comment by thewongdev** [comment]
Just circling back to this. Set up a new mac mini and and loaded up Quiver from dropbox backup. Had to manually set all my favorites again. Please consider adding favs to sync!
